### PR TITLE
Should address all the comments we discussed in the PR

### PR DESCRIPTION
**Pull request by ElDeveloper**

### CODE REVIEW

File: q2_qemistree/_itol_metadata.py
Comment (anupriyatripathi) [inline q2_qemistree/_itol_metadata.py:54]: looks great!

File: q2_qemistree/_prune_hierarchy.py
Comment (anupriyatripathi) [inline q2_qemistree/_prune_hierarchy.py]: Is this missing some text?

File: q2_qemistree/_prune_hierarchy.py
Comment (anupriyatripathi) [inline q2_qemistree/_prune_hierarchy.py:54]: This should be very useful -- thanks for adding!

File: q2_qemistree/_prune_hierarchy.py
Comment (anupriyatripathi) [inline q2_qemistree/_prune_hierarchy.py:62]: 👍

File: q2_qemistree/tests/test_classyfire.py
Comment (anupriyatripathi) [inline q2_qemistree/tests/test_classyfire.py]: It should be fine for now - thanks for adding the TODO.

File: q2_qemistree/tests/test_prune_hierarchy.py
Comment (anupriyatripathi) [inline q2_qemistree/tests/test_prune_hierarchy.py:29]: :D

File: q2_qemistree/_itol_metadata.py
Comment (anupriyatripathi) [inline q2_qemistree/_itol_metadata.py]: I think this should be:
```suggestion
warnings.warn("The mapping between colors and annotations"
              " is not unique, some colors have been repeated", UserWarning)
```

**Review by anupriyatripathi — CHANGES_REQUESTED** [review]
Hi Yoshiki, the fixes look really good. I just have a couple of minor edit requests which were probably typos after which it should be ready to merge! I can update the README if you prefer. Thanks!

File: q2_qemistree/_prune_hierarchy.py
Comment (ElDeveloper) [reply q2_qemistree/_prune_hierarchy.py]: Fixed, thanks for catching this.

File: q2_qemistree/tests/test_classyfire.py
Comment (ElDeveloper) [reply q2_qemistree/tests/test_classyfire.py]: Thanks for catching this, I just added a test

**Review by anupriyatripathi — APPROVED** [review]
Looks great - thanks!

**Pull request merged**
